yarn.lock

```diff
@@ -1534,26 +1388,23 @@
 "@lerna/timer@3.13.0":
   version "3.13.0"
   resolved "https://registry.yarnpkg.com/@lerna/timer/-/timer-3.13.0.tgz#bcd0904551db16e08364d6c18e5e2160fc870781"
-  integrity sha512-RHWrDl8U4XNPqY5MQHkToWS9jHPnkLZEt5VD+uunCKTfzlxGnRCr3/zVr8VGy/uENMYpVP3wJa4RKGY6M0vkRw==
 
 "@lerna/validation-error@3.13.0":
   version "3.13.0"
   resolved "https://registry.yarnpkg.com/@lerna/validation-error/-/validation-error-3.13.0.tgz#c86b8f07c5ab9539f775bd8a54976e926f3759c3"
-  integrity sha512-SiJP75nwB8GhgwLKQfdkSnDufAaCbkZWJqEDlKOUPUvVOplRGnfL+BPQZH5nvq2BYSRXsksXWZ4UHVnQZI/HYA==
```

Comment:
Why is all the integrity data being removed ?

Reply:
My bad, i deleted the yarn.lock on a test that i was doing... I will restore this file

Reply:
Ok, i was looking and when i pushed the other changes, it was not passing the tests on CircleCi, it was because its running yarn with lock files (I think thats because when you yarn install, that can take some time).
So i re-upload the yarn.lock again